modules/ROOT/pages/prepare-to-install-service-mesh.adoc

```diff
@@ -100,6 +100,65 @@ For example, type `cd /home/_your-user-name_/.bashrc`) in a text editor.
 . Save the `.bashrc` file.
 . Restart your terminal.
 
+== Preparing to Install Instio
+
+When you install Instio in your environment, you must configure Instio settings based on the Instio version that you install.
+
+=== Installing Instio 1.4x
```

Comment:
```suggestion
=== Installing Istio 1.4.x
```

Reply:
GLOBALLY search for this version typo as well.

Reply:
Done! Thanks!